Libraries/dotNetRdf.Core/Query/Algebra/Extend.cs

```diff
@@ -95,14 +95,14 @@ public IEnumerable<string> Variables
     {
         get 
         {
-            return InnerAlgebra.Variables.Concat(VariableName.AsEnumerable()); 
+            return [.. InnerAlgebra.Variables, VariableName]; 
```

Comment:
Using a [spread element](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/collection-expressions#spread-element) to simplify concatenation.